src/game/stats.c

```diff
@@ -382,7 +382,7 @@ void Stats_Show(int32_t level_num)
     g_GameInfo.status &= ~GMS_IN_STATS;
 }
 
-void Stats_ShowTotal(const char *filename)
+void Stats_ShowTotal(const char *filename, int first_level, int last_level)
```

Comment:
Is it worth adding a way to show "Bonus Statistics" as the title here? So that'd be another string in the gameflow to allow builders to customize it. I know we discussed it on Discord, I'm just remembering about it now.